app/models/concerns/galtersufia/generic_file/license_data.txt

```diff
@@ -9,31 +9,31 @@
   },
```

Comment:
might as well downcase `GALTER-ARR-1.0` to `galter-arr-1,0` and `CC-PDM-1.0` to `cc-pdm-1.0` . I will do the changes on digitalhub-2.